conda/core/initialize.py

```diff
@@ -1201,7 +1429,31 @@ def _config_fish_content(conda_prefix):
     return conda_initialize_content
 
 
-def init_fish_user(target_path, conda_prefix, reverse):
+def _config_fish_content_to_add_condabin_to_path(conda_prefix):
+    condabin_dir = join(conda_prefix, "condabin")
+    if on_win:
+        from ..activate import native_path_to_unix
+
+        condabin_dir = native_path_to_unix(condabin_dir)
```

Comment:
This has been deprecated:

```suggestion
        from ..common.path import win_path_to_unix

        condabin_dir = win_path_to_unix(condabin_dir)
```